src/management/index.js

```diff
@@ -3803,4 +3811,26 @@ utils.wrapPropertyMethod(
   'prompts.updateCustomTextByLanguage'
 );
 
+/**
+ * Get organizations
+ *
+ * @method    getOrganizations
+ * @memberOf  module:management.ManagementClient.prototype
+ *
+ * @example
+ * management.getOrganizations(function (err, organizations) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ *
+ * // Migration flags
+ *    console.log(organizations.flags);
+ * });
+ *
+ * @param   {Function}  [cb]              Callback function.
+ *
+ * @return    {Promise|undefined}
+ */
+utils.wrapPropertyMethod(ManagementClient, 'getOrganizations', 'organizations.getAll');
```

Comment:
I didn't get why we have `getOrganizations` twice. Here and [here](https://github.com/auth0/node-auth0/pull/592/files#diff-568c4929c14a5fc0b13ddf873c7646a74e71f600f40976d911fec2fef3494daaR140). 🤔